### DISCUSSION TITLE
Space between words on RTL

### DISCUSSION THREAD

**Discussion by vivekpshaligram** [readium/kotlin-toolkit#579] [Q&A]
Could any one provide any additional references or solutions for adjusting word-spacing in RTL layout?

As it's mention it works on LTR layout only.

`/**
     * Space between words.
     *
     * Only effective when:
     *  - the publication is reflowable
     *  - the layout is LTR
     */
    val wordSpacing: RangePreference<Double> =
        RangePreferenceDelegate(
            getValue = { preferences.wordSpacing },
            getEffectiveValue = { state.settings.wordSpacing ?: 0.0 },
            getIsEffective = ::isWordSpacingEffective,
            updateValue = { value -> updateValues { it.copy(wordSpacing = value) } },
            supportedRange = 0.0..1.0,
            progressionStrategy = DoubleIncrement(0.1),
            valueFormatter = percentFormatter(),
        )`

**Comment by Shtibel** [comment]
Yes, this will be very helpful to me too !

**Comment by mickael-menu** [comment]
Could you open this issue on https://github.com/readium/readium-css instead? We rely on this component for that.
